apps/artist/client/views/header.coffee

```diff
@@ -31,6 +31,7 @@ module.exports = class ArtistHeaderView extends Backbone.View
         following: view.following
         modelName: 'artist'
         model: view.model
+        authOnClick: !SHOW_ARTIST_CTA
```

Comment:
yea, that just will pass `true`, which is also the default behavior (if you didn't pass anything).